frappe/desk/page/setup_wizard/install_fixtures.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+
+import frappe
+
+from frappe import _
+
+def install():
+	update_genders_and_salutations()
+
+@frappe.whitelist()
+def update_genders_and_salutations():
+	default_genders = [_("Male"), _("Female"), _("Other")]
+	default_salutations = [_("Mr"), _("Ms"), _('Mx'), _("Dr"), _("Mrs"), _("Madam"), _("Miss"), _("Master"), _("Prof")]
+	records = [{'doctype': 'Gender', 'gender': d} for d in default_genders]
+	records += [{'doctype': 'Salutation', 'salutation': d} for d in default_salutations]
+	for record in records:
+		doc_name = record.get("doctype")
```

Comment:
Tested and removed the if exist code, not required since we are checking DuplicateEntryError